libhttpserver/server.go

```diff
@@ -103,7 +117,12 @@ func (s *Server) getHTTPFileSystem(ctx context.Context, requestPath string) (
 		return "", nil, err
 	}
 
-	s.fs.Add(toStrip, tlfFS)
+	fsLifeCh, err := tlfFS.SubscribeToEndOfLife()
```

Comment:
I'd prefer something like `SubscribeToShutdown()` here

Reply:
It's not a shutdown though; as it can still be used.

Reply:
Also since this isn't the same as shutting down a folder branch ops, and we already have `Shutdown` methods on various types, I think it'd be better to name this one differently to avoid confusion.

Reply:
It means user of this type shouldn't use this anymore (as mentioned on the method comment). I was trying to avoid mentioning "handle change" as that's a concept that the `libfs` layer hasn't had to expose yet. If end-of-life doesn't make sense here, I'd gladly consider other names. Would `SubscribeToObsolete` be better?

Reply:
Oh yeah that would make more sense to me.